recipes/py_mini_racer/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "py_mini_racer" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/py_mini_racer-{{ version }}.tar.gz
+  sha256: f71e36b643d947ba698c57cd9bd2232c83ca997b0802fc2f7f79582377040c11
+
+build:
+  skip: True
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  build:
+    - libc-musl
+    - libstdcxx-ng
```

Comment:
Why do you need these?

Reply:
Dependencies that are necessary to run on linux.
And sorry for the wrong ping!

Reply:
Oh, these are anaconda dependencies...
I guess I just won't build to linux then.
I'll update this later today.